config.json

```diff
@@ -1267,7 +1280,13 @@
         "name": "Simple Linked List",
         "uuid": "a1591026-2f02-45f9-b189-24b2359eb43f",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "arrays",
+          "classes",
+          "conditionals",
+          "increment-decrement",
+          "while-loops"
```

Comment:
* Here we need either `for-loops` or `array-loops` because the student needs to iterate over the argument (array) that was passed in to transform them to the nodes/elements. You wouldn't use a while loop here.
* I would remove `increment-decrement`, the exercise can be solved without it and I plan to integrate that concept with `arithmetic-operators` in the future.